### ISSUE TITLE
Is it possible to get a player hit statistic

### ISSUE THREAD

**Issue by DFSCompad** [schroeder-lvb/sissm#81]
Read the log file and write it to mysql database. Player is wunded right arm or something like that.

**Comment by schroeder-lvb** [comment]
Yes it is under consideration but on a very low priority list.  As far as I can tell SISSM only runs on handful of servers so I prioritize on features for which SISSM was designed:  automated server watchdog/management.  I've also slowed down development pending SS v1.5 release to evaluate what is under-the-hood with the mod-kit, map-kit, which may impact current and future SISSM scope and features.

**Comment by schroeder-lvb** [comment]
Engine.ini segment
================================
```
[Core.Log]
LogGameplayEvents=Display
LogDemo=Display
LogObjectives=Verbose
LogGameMode=Display
```


Sample Insurgency.log: Enter/Exit Territorial Objective Zone or Weapons Cache Proximity
================================

`[2020.03.04-22.23.22:953][ 19]LogObjectives: Verbose: Objective ODCheckpoint_A: BP_Character_Player_C_5 entered.

[2020.03.04-22.23.40:564][ 70]LogObjectives: Verbose: Objective ODCheckpoint_A: BP_Character_Player_C_5 exited.`

Sample Insurgency.log: Kills/Deaths
================================

[2020.03.04-22.22.44:423][722]LogGameplayEvents: Display: YourPlayerName[76560000000000000, team 0] killed Demolitions[INVALID, team 1] with BP_Firearm_QBZ03_C_2

[2020.03.05-00.04.51:027][827]LogGameplayEvents: Display: YourPlayerName1[76560000000000001, team 0] + YourPlayerName2[76560000000000002, team 0] killed Breacher[INVALID, team 1] with BP_Firearm_MP5A2_C_1

Sample Insurgency.log: Destructive Cache "Destroyed"
================================

[2020.03.04-22.23.08:786][173]LogGameplayEvents: Display: Objective 0 owned by team 1 was destroyed for team 0 by YourPlayerName[76560000000000000].

[2020.03.05-00.19.24:869][525]LogGameplayEvents: Display: Objective 1 owned by team 1 was destroyed for team 0 by YourPlayerName6[76560000000000006], YourPlayerName7[76560000000000007], YourPlayerName5[76560000000000005].

[2020.03.05-00.35.45:360][277]LogGameplayEvents: Display: Objective 4 owned by team 1 was destroyed for team 0 by YourPlayerName6[76560000000000006], YourPlayerName7[76560000000000007], YourPlayerName1[76560000000000001], YourPlayerName4[76560000000000004], YourPlayerName5[76560000000000005].



Sample Insurgency.log: Territorial Zone "Captured"
================================

[2020.03.04-22.31.09:488][843]LogGameplayEvents: Display: Objective 2 was captured for team 0 from team 1 by YourPlayerName[76560000000000000].

[2020.03.05-00.10.08:782][619]LogGameplayEvents: Display: Objective 3 was captured for team 0 from team 1 by YourPlayerName3[76560000000000003], YourPlayerName1[76560000000000001].

[2020.03.05-00.13.00:194][775]LogGameplayEvents: Display: Objective 4 was captured for team 0 from team 1 by YourPlayerName1[76560000000000001], YourPlayerName3[76560000000000003], YourPlayerName8[76560000000000008].
